Checkstyle issues: Avoid inline conditionals.

The below locations (file name:line number:character number) were flagged with "Avoid inline conditionals." in Checkstyle:

	Line 1: JTAF-XCore\src\main\java\org\finra\jtaf\core\asserts\AssertionFailedError.java:34:40: error: Avoid inline conditionals.
	Line 1995: JTAF-XCore\src\main\java\org\finra\jtaf\core\model\test\TestAgenda.java:60:66: error: Avoid inline conditionals.
	Line 2001: JTAF-XCore\src\main\java\org\finra\jtaf\core\model\test\TestAgenda.java:72:67: error: Avoid inline conditionals.
	Line 2006: JTAF-XCore\src\main\java\org\finra\jtaf\core\model\test\TestAgenda.java:81:49: error: Avoid inline conditionals.
	Line 4392: JTAF-XCore\src\main\java\org\finra\jtaf\core\utilities\CompositeDataComparator.java:132:65: error: Avoid inline conditionals.

--- a/pom.xml
+++ b/pom.xml
@@ -3,7 +3,7 @@
 	<modelVersion>4.0.0</modelVersion>
 	<groupId>org.finra.jtaf</groupId>
 	<artifactId>jtaf-xcore</artifactId>
-	<version>1.1-SNAPSHOT</version>
+	<version>1.2-SNAPSHOT</version>
 	<packaging>jar</packaging>
 	<name>JTAF XCore</name>
 	<description>The core functionality provided by JTAF. Includes script parsing, test execution, and JUnit support.</description>

--- a/pom.xml
+++ b/pom.xml
@@ -3,7 +3,7 @@
 	<modelVersion>4.0.0</modelVersion>
 	<groupId>org.finra.jtaf</groupId>
 	<artifactId>jtaf-xcore</artifactId>
-	<version>1.1</version>
+	<version>1.1-SNAPSHOT</version>
 	<packaging>jar</packaging>
 	<name>JTAF XCore</name>
 	<description>The core functionality provided by JTAF. Includes script parsing, test execution, and JUnit support.</description>

--- a/src/main/java/org/finra/jtaf/core/asserts/AssertionFailedError.java
+++ b/src/main/java/org/finra/jtaf/core/asserts/AssertionFailedError.java
@@ -31,6 +31,10 @@ public class AssertionFailedError extends AssertionError {
 	}
 
 	private static String defaultString(String message) {
-		return message == null ? "" : message;
+		if (message == null) {
+			return "";
+		} else {
+			return message;
+		}
 	}
 }

--- a/src/main/java/org/finra/jtaf/core/model/test/TestAgenda.java
+++ b/src/main/java/org/finra/jtaf/core/model/test/TestAgenda.java
@@ -57,8 +57,11 @@ public class TestAgenda {
 	 * @return read only set of automation values
 	 */
 	public Set<String> getAutomationValues() {
-		Set<String> setToWrap = automationValues == null ? (new HashSet<String>()) : automationValues;
-		return Collections.unmodifiableSet(setToWrap);
+		if (automationValues == null) {
+			return Collections.emptySet();
+		} else {
+			return Collections.unmodifiableSet(automationValues);
+		}
 	}
 	
 	/**
@@ -69,7 +72,11 @@ public class TestAgenda {
 	 * @return true if value exists, case insensitive
 	 */
 	public boolean containsAutomationValue(String value) {
-		return value != null && (automationValues == null ? false : automationValues.contains(value.toLowerCase()));
+		if (value == null || automationValues == null) {
+			return false;
+		} else {
+			return automationValues.contains(value.toLowerCase());
+		}
 	}
 	
 	/**
@@ -78,7 +85,7 @@ public class TestAgenda {
 	 * @return true if null or empty
 	 */
 	public boolean isAutomationValuesEmpty() {
-		return automationValues == null ? true : automationValues.isEmpty();
+		return automationValues == null || automationValues.isEmpty();
 	}
 
 	public void setThreadCount(String threads){

--- a/src/main/java/org/finra/jtaf/core/utilities/CompositeDataComparator.java
+++ b/src/main/java/org/finra/jtaf/core/utilities/CompositeDataComparator.java
@@ -129,8 +129,11 @@ public class CompositeDataComparator {
 	}
 
 	private String getPrefix(String title, String element) {
-		return (element == null && !"".equals(element)) ? title : title + "-> "
-				+ element + " : ";
+		if (element == null && !"".equals(element)) {
+			return title;
+		} else {
+			return title + "-> " + element + " : ";
+		}
 	}
 
 	private void assessAndCompare(String title, String element,

